packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -72,6 +73,15 @@ export interface VideoGalleryStrings {
   localVideoCameraSwitcherLabel: string;
 }
 
+/**
+ * @public
+ */
+export type LocalTileMode =
+  | 'default'
+  | 'floatingLocalVideo'
+  | /* @conditional-compile-remove-from(stable) meeting/calling-composite <Local-Camera-Switcher> */ 'controlsLocalVideo'
+  | undefined;
```

Comment:
The type shouldn't include `undefined`. Instead, make the field that uses this type optional. (It actually already is optional `VideoGallyerProps.layout`)

Reply:
okay will remove this!